openquake.cfg

```diff
@@ -41,7 +41,8 @@ celery_queue = celery
 [dbserver]
 # enable multi_user if you have a multiple user installation
 multi_user = false
-file = ~/db.sqlite3
+file = ~/oqdata/db.sqlite3
+log = ~/oqdata/dbserver.log
```

Comment:
Moving these files into `oqdata` will led to an error if oqdata does not exist yet:

```IOError: [Errno 2] No such file or directory: '/home/dvigano/oqdata/dbserver.log'```